### DISCUSSION TITLE
生成了可用的语音pdz，但是不知道怎么调用

### DISCUSSION THREAD

**Discussion by yunqi777** [PaddlePaddle/PaddleSpeech#2595] [Q&A]
俺是一个纯萌新，通过照搬官方的这个项目↓↓↓（这个项目是我目前看到最适合新手的了）
https://aistudio.baidu.com/aistudio/projectdetail/4652235
我达到了比较好的克隆效果，但是每次项目重启项目环境发生改变，就需要重头运行整个项目才能再次合成
原本这没什么，但是由于我的数据集有五千条语音，所以每次合成都要花一个半小时QWQ
不过其实之前的模型并没有消失，pdz文件已经生成，只是因为项目需要一个特定的变量因此无法运行
![image](https://user-images.githubusercontent.com/110005064/198597846-46e10656-7814-4fa2-8dbc-fa15a35de5c5.png)
有没有办法能够直接调用原先生成的文件，而不需要再次生成呢？
麻烦诸位大佬了！！！

**Comment by yt605155624** [comment]
find_max_ckpt() 函数应该是找到目录里面最新生成的 ckpt (应该是标号最大的 ckpt)，所以如果目录中有 pdz 文件应该是可以用的，或许你可以直接看源码 https://github.com/PaddlePaddle/PaddleSpeech/tree/develop/examples/other/tts_finetune/tts3 源码中是自己指定 ckpt_name
